site/content/docs/5.2/examples/cards/index.html

```diff
@@ -93,89 +95,88 @@ <h6 class="card-text">Caption</h6>
         </div>
       </div>
 
-      <div class="col m-0 py-2">
+      <div class="col">
         <div class="card border-0">
           {{< placeholder width="100%" height="162" class="card-img-top" text="Image cap" >}}
           <div class="card-body bg-dark">
-            <h4 class="card-title">Title</h4>
-            <h6 class="card-text">Caption</h6>
+            <h5 class="card-title">Title</h5>
+            <p class="card-text fw-bold mb-2">Caption</p>
           </div>
-          <div class="card-footer bg-dark text-light pt-2">
-            <a class="btn btn-secondary" aria-label="Button description" title="Button description">Button</a>
+          <div class="card-footer bg-dark">
```

Comment:
I think there are modifications to do in this file (and in the corresponding RTL file) to remove the `.card-footer` since we finally haven't added it in the Cards component page.